docs/iris/src/developers_guide/pulls.rst

```diff
@@ -62,20 +63,20 @@ The Iris Check List
 
  * ``make doctest``, ``make extest``  in ``./docs/iris``.
```

Comment:
It seems like ``make extest`` isn't actually a thing, but before I remove it does anyone know what, if anything, replaces it and if it's worth including here?